src/runtime_src/core/include/xgq_cmd_ert.h

```diff
@@ -193,10 +193,10 @@ struct xgq_cmd_config_end {
  *
  * @cu_idx: cu index to configure
  * @ip_ctrl: IP control protocol
- * @regmap_size: the maximum CU register map size
+ * @map_size: use this size to map CU if apply
  * @laddr: lower 32 bits of the CU address
  * @haddr: higher 32 bits of the CU address
- * @real_size: the minimum CU register map size includes the last CU argument
+ * @pl_size: CU XGQ slot payload size
```

Comment:
let's spell out "payload" here ("payload_size"). it is better readable in the code.